routes/review.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express')
+
+const idValidation = require('~/middlewares/idValidation')
+const asyncWrapper = require('~/middlewares/asyncWrapper')
+const reviewController = require('~/controllers/review')
+
+const router = express.Router()
+
+router.param('reviewId', idValidation)
```

Comment:
mby we should stick to one format? just id? for every model? :id

Reply:
ok